### PR TITLE
add `Vishnu Ks`

### PR DESCRIPTION
**Pull request by hackerkid**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ Amas is a [recursive acronym](https://en.wikipedia.org/wiki/Recursive_acronym).
 - [Blain Smith](https://github.com/blainsmith/ama) - Partner/Engineer at MadGlory and Odd Networks. Rebel Geek.
 - [Paul Miller](https://github.com/paulmillr/ama) - Creator of Brunch.io, Chaplin, es6-shim and many npm modules. Entrepreneur & frequent traveler.
 - [Lewis Cowper](https://github.com/lewiscowper/ama) - Former forklift driver. Now web developer.
+- [Vishnu Ks](https://github.com/hackerkid/ama) - Experimentalist
```
Comment (sindresorhus) [inline readme.md]: Can you come up a slightly longer description? Also needs a dot at the end.

**Comment by sindresorhus** [comment]
Merge conflict needs to be fixed.

**Comment by hackerkid** [comment]
I have created a new pull request. [#24](https://github.com/sindresorhus/amas/pull/24)

**Comment by arthurvr** [comment]
Please don't create new pull requests, just update the existing one using a
rebase.

**Pull request closed without merging**